doc/linux.t2t

```diff
@@ -210,10 +210,39 @@ Build and install with ninja:
 ninja (uses all cores by default; also supports the above described -jX option)
 ninja install
 ```
-You can build just the targets you need using, for example:
+
+To build even faster, you can build just the targets you need using, for example:
 ```
 ninja qgis
 ninja pycore
+# if it's on desktop related code only:
+ninja qgis_desktop
+```
+
+== Compiling with 3D ==
+
+QGIS 3D requires Qt53DExtras. Unfortunately, these headers have been removed from Qt upstream.
```

Comment:
This isn't needed everywhere -- only on debian based systems

Reply:
MacOS too